test/prob/generate_tests.cpp

```diff
@@ -25,44 +25,37 @@ vector<string> lookup_argument(const string& argument, const int& ind) {
     args.push_back("int");
     args.push_back("std::vector<int>");
     args.push_back("Eigen::Matrix<int, Eigen::Dynamic, 1>");
-    args.push_back("Eigen::Matrix<int, 1, Eigen::Dynamic>");
```

Comment:
I didn't even notice this. We should remove lines 27-28. We don't instantiate `Eigen::Matrix<int, *, *>` from the Stan language. I don't see any reason someone would from within the math C++ either. (We must have done it for completeness.)

Reply:
Sweet. For now I'm going to leave it in because from what I can tell, the number of tests is set statically independent of the types and whatnot, so it wouldn't work to have differently sized lists here:
```
364     num_tests = size(argument_sequence) - std::pow(3,num_ints(arguments))        
365       * std::pow(3, num_doubles(arguments));      
```

I'll remove it in the rewrite rather than trying to refactor this code.